### PR TITLE
Update security.md

### PR DESCRIPTION
**Pull request by CorruptedCas**
Updated the security resource page, adding information about the new sessions/devices feature and correcting some slight errors.

### CODE REVIEW

**Review by priyanujgogoi-28 — COMMENTED** [review]
I'll try to review this later.

File: src/resources/security.md [hunk 1]
```diff
@@ -3,17 +3,31 @@ Security is an important topic to discuss. If security measures are disregarded,
 
 This article will share tips about how you can keep your bot and account safe.
 
-### Sharing Tokens
+### Sharing Tokens and Passwords
 **Do not share token(s) with anyone**. This includes both bot and regular user account tokens. Sharing your bot token with someone (or posting it publicly) grants them full-unlimited access to your editing bot. Sharing your user account token with someone (or posting it publicly) allows them to have full access to your account (even if they don't have your password or email). Once someone has your account or bot's token, there is a high chance of it being used for malicious purposes. For example, stealing personal info, spreading scams, modifying your bot to nuke/raid servers.
```
Comment (priyanujgogoi-28) [inline src/resources/security.md]:
```suggestion
**Do not share token(s) with anyone**. This includes both bot and regular user account tokens. Sharing your bot token with someone (or posting it publicly) will grant them full access to edit your bot. While, sharing your user account token with someone (or posting it publicly) will allow them to have full access to your account (even if they don't have your password or email). Once someone has your account or bot's token, there is a high chance of it being used for malicious purposes. For example, stealing personal info, spreading scams, modifying your bot to nuke/raid servers etc.
```

File: src/resources/security.md [hunk 2]
```diff
@@ -3,17 +3,31 @@ Security is an important topic to discuss. If security measures are disregarded,
...
 
-The only thing you can do is regenerate your bot's token, but likely, the damage has already been done when you regenerate the token. You can't regenerate user account tokens, in such case you need to; you must contact Discord support for an optimal solution. So please, do not risk sharing your token(s) with anyone.
-The same applies to your account password, in such case you expose your account password; you should change it in user settings as soon as possible.
+In the event that your **bot's token** is shared, the only thing you can do is regenerate it, but by then, the damage has most likely been done. In the event you share your **account token** but still have access to your account, you can regenerate your token by changing your password. If you don't, you will need to contact Discord support for an optimal solution.
```
Comment (priyanujgogoi-28) [inline src/resources/security.md]:
```suggestion
In an event, if your **bot's token** is shared, the only thing you can do to secure it, is to regenerate a new bot token but by then, the damage has most likely been done. In the case of an **user account token**, if you still have access to your account, regenerate your token by changing your password. If you cannot or don't have access to it anymore, you will need to contact [Discord support](https://support.discord.com/hc/en-us/requests/new) for an optimal solution.
```

File: src/resources/security.md [hunk 3]
```diff
@@ -3,17 +3,31 @@ Security is an important topic to discuss. If security measures are disregarded,
...
 Bot owners should consider enabling two-factor authentication on their accounts. [Learn more about 2FA and why it's essential for bot owners](./2FA.md).
 
+### Sessions
+Discord recently added the ability to see all your current sessions and their respective locations.
+
+![image](https://user-images.githubusercontent.com/116581988/199214548-911dc935-3c52-43cc-b0d7-74d18e283903.png)
+
+If you see a device or location that you haven't authorised, you can log out of that particular device by pressing the 'X' button or all known devices by clicking the button at the bottom of the page. This will log out those sessions, invalidating the tokens. 
+
+![image](https://user-images.githubusercontent.com/116581988/199320324-fd93384f-31e1-4a50-98c3-f4f875b91536.png)
```
Comment (priyanujgogoi-28) [inline src/resources/security.md]:
```suggestion
![image](https://user-images.githubusercontent.com/116581988/199320324-fd93384f-31e1-4a50-98c3-f4f875b91536.png)\
```

File: src/resources/security.md [hunk 4]
```diff
@@ -3,17 +3,31 @@ Security is an important topic to discuss. If security measures are disregarded,
...
+
```
Comment (priyanujgogoi-28) [inline src/resources/security.md]:
```suggestion
```

File: src/resources/security.md [hunk 5]
```diff
@@ -3,17 +3,31 @@ Security is an important topic to discuss. If security measures are disregarded,
...
+![image](https://user-images.githubusercontent.com/116581988/199320198-e856f6f2-47dc-4dda-bd8c-809eb34bcfdb.png)
```
Comment (priyanujgogoi-28) [inline src/resources/security.md]:
```suggestion
![image](https://user-images.githubusercontent.com/116581988/199320198-e856f6f2-47dc-4dda-bd8c-809eb34bcfdb.png)\
```

File: src/resources/security.md [hunk 6]
```diff
@@ -3,17 +3,31 @@ Security is an important topic to discuss. If security measures are disregarded,
...
+
```
Comment (priyanujgogoi-28) [inline src/resources/security.md]:
```suggestion
```

**Review by priyanujgogoi-28 — CHANGES_REQUESTED** [review]

File: src/resources/security.md (hunk 2 above)
Comment (MineBartekSA) [reply src/resources/security.md]: This line is quite long, it should be split.
```suggestion
In the event, that your **bot's token** is shared, the only thing you can do to secure it is to regenerate the bot's token. But by then, the damage has most likely been done.
In the case of a **user account token**, if you still have access to your account, regenerate your token by changing your password. If you cannot or don't have access to it anymore, you will need to contact [Discord support](https://support.discord.com/hc/en-us/requests/new) for an optimal solution.
```

File: src/resources/security.md (hunk 1 above)
Comment (MineBartekSA) [reply src/resources/security.md]: No need for such a long line.
```suggestion
**Do not share token(s) with anyone**. This includes both bot and regular user account tokens. Sharing your bot token with someone (or posting it publicly) will grant them full access to edit your bot.
While sharing your user account token with someone (or posting it publicly) will allow them to have full access to your account (even if they don't have your password or email).
Once someone has your account or bot's token, there is a high chance of it being used for malicious purposes. For example, stealing personal info, spreading scams, modifying your bot to nuke/raid servers, etc.
```

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

**Review by priyanujgogoi-28 — CHANGES_REQUESTED** [review]
Screenshot_20221102_145030

In my opinion, the above statement should be instead, _“This sub-section will breakdown how you can identify an untrusted link and a trusted link.”_

File: src/resources/security.md [hunk 7]
```diff
@@ -3,25 +3,40 @@ Security is an important topic to discuss. If security measures are disregarded,
...
-The same applies to your account password, in such case you expose your account password; you should change it in user settings as soon as possible.
+In the event, that your **bot's token** is shared, the only thing you can do to secure it is to regenerate the bot's token. But by then, the damage has most likely been done.
+In the case of a **user account token**, if you still have access to your account, regenerate your token by changing your password. If you cannot or don't have access to it anymore, you will need to contact [Discord support](https://support.discord.com/hc/en-us/requests/new) for an optimal solution.
+
+Passwords, like tokens, should not be shared. If, however, you accidentally share your account password, you should change it as soon as possible.
 
 > 📝 If your account is hacked, you should [contact Discord](https://support.discord.com/hc/requests/new) for further assistance.
```
Comment (priyanujgogoi-28) [inline src/resources/security.md]:
```suggestion
> 📌 If your account is hacked, you should [contact Discord](https://support.discord.com/hc/requests/new) for further assistance.
```

**Review by priyanujgogoi-28 — COMMENTED** [review]
Well, i would be glad if you could change this emoji as well!

**Review by priyanujgogoi-28 — APPROVED** [review]
Thanks!

**Review by MineBartekSA — APPROVED** [review]
Thank you for your hard work

**Pull request merged**